### PR TITLE
Fix "SVC failure due to adding strict types" section of release notes

### PR DESCRIPTION
**Pull request by ankurvr**
## Purpose of this pull request

This pull request (PR) is to make docs more clean and fix the irrelevant descriptions.

## Affected DevDocs pages
https://devdocs.magento.com/guides/v2.4/release-notes/backward-incompatible-changes/index.html

-  ...

## Links to Magento source code
Not exactly related to source code.

-  ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.4/release-notes/backward-incompatible-changes/index.md [hunk 1]
```diff
@@ -238,13 +238,13 @@ MAJORMagento\Catalog\Helper\Product\ProductList::getAvailableLimit /app/code/Mag
 Return type now array (the same as before in DocBlock):
 
 ```terminal
-changed.MAJORMagento\Catalog\Helper\Product\ProductList::getAvailableLimit /app/code/Magento/Catalog/Helper/Product/ProductList.php:122V060 [public] Method parameter name
+changed.MAJOR Magento\Catalog\Helper\Product\ProductList::getAvailableLimit /app/code/Magento/Catalog/Helper/Product/ProductList.php:122V060 [public] Method parameter name
```
Comment (dmytro-ch) [inline src/guides/v2.4/release-notes/backward-incompatible-changes/index.md:241]:
```suggestion
changed.MAJOR: Magento\Catalog\Helper\Product\ProductList::getAvailableLimit /app/code/Magento/Catalog/Helper/Product/ProductList.php:122V060 [public] Method parameter name
```

File: src/guides/v2.4/release-notes/backward-incompatible-changes/index.md [hunk 2]
```diff
@@ -238,13 +238,13 @@ MAJORMagento\Catalog\Helper\Product\ProductList::getAvailableLimit /app/code/Mag
...
 ```
 
 Now returns int for `DefaultLimitPerPageValue`:
 
 ```terminal
-changed.MAJORMagento\Catalog\Helper\Product\ProductList::getDefaultLimitPerPageValue /app/code/Magento/Catalog/Helper/Product/ProductList.php:147M120 [public] Method return typing changed.
+changed.MAJOR Magento\Catalog\Helper\Product\ProductList::getDefaultLimitPerPageValue /app/code/Magento/Catalog/Helper/Product/ProductList.php:147M120 [public] Method return typing changed.
```
Comment (dmytro-ch) [inline src/guides/v2.4/release-notes/backward-incompatible-changes/index.md:247]:
```suggestion
changed.MAJOR: Magento\Catalog\Helper\Product\ProductList::getDefaultLimitPerPageValue /app/code/Magento/Catalog/Helper/Product/ProductList.php:147M120 [public] Method return typing changed.
```

File: src/guides/v2.4/release-notes/backward-incompatible-changes/index.md [hunk 3]
```diff
@@ -238,13 +238,13 @@ MAJORMagento\Catalog\Helper\Product\ProductList::getAvailableLimit /app/code/Mag
 Return type now array (the same as before in DocBlock):
```
Comment (dmytro-ch) [inline src/guides/v2.4/release-notes/backward-incompatible-changes/index.md:238]:
```suggestion
The input parameter is renamed to `viewMode`:
```

File: src/guides/v2.4/release-notes/backward-incompatible-changes/index.md (hunk 1 above)
Comment (ankurvr) [reply src/guides/v2.4/release-notes/backward-incompatible-changes/index.md]: Done

File: src/guides/v2.4/release-notes/backward-incompatible-changes/index.md (hunk 3 above)
Comment (ankurvr) [reply src/guides/v2.4/release-notes/backward-incompatible-changes/index.md]: Done

File: src/guides/v2.4/release-notes/backward-incompatible-changes/index.md (hunk 2 above)
Comment (ankurvr) [reply src/guides/v2.4/release-notes/backward-incompatible-changes/index.md]: Dobe

**Comment by dobooth** [comment]
running tests

**Pull request merged**